### PR TITLE
[BB-714] Add API for removing Theme and delete Theme images on delete

### PR DESCRIPTION
**Pull request by Agrendalath**
Please see instructions in the private PR.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Agrendalath! I've created [OSPR-2978](https://openedx.atlassian.net/browse/OSPR-2978) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: mobileapps/views.py [hunk 1]
```diff
@@ -1017,6 +1017,9 @@ class OrganizationThemeDetailView(MobileRetrieveUpdateDestroyAPIView):
         * navigation_icon_color
         * completed_course_tint
         * lesson_navigation_color
+
+        **DELETE**
+        Deactivates the theme by default. Set query param `remove` to `true` to actually theme and its images.
```
Comment (giovannicimolin) [inline mobileapps/views.py]: Shouldn't this be: `Set query param remove to true to actually delete theme and its images.`?

File: mobileapps/views.py [hunk 2]
```diff
@@ -1149,7 +1157,7 @@ def __init__(self):
         self.permission_classes += (IsStaffView,)
 
     @transaction.atomic
-    def delete(self, request, theme_id, attribute):
+    def delete(self, _request, theme_id, attribute):
```
Comment (giovannicimolin) [inline mobileapps/views.py:1160]: Doesn't this view require any special permissions? Can anyone with API access make a request to this?

**Review by giovannicimolin — CHANGES_REQUESTED** [review]

File: mobileapps/views.py (hunk 2 above)
Comment (Agrendalath) [reply mobileapps/views.py:1160]: There is `IsStaffOrReadOnlyView`. Please see [this line](https://github.com/edx-solutions/mobileapps-edx-platform-extensions/blob/9262213a3beee88ccbaeaccd2fe9bc087f5dcf42/mobileapps/views.py#L1029)

Comment (giovannicimolin) [reply mobileapps/views.py:1160]: I missed that.

File: mobileapps/views.py (hunk 1 above)
Comment (Agrendalath) [reply mobileapps/views.py]: Of course it should :)
Fixed!

**Review by giovannicimolin — APPROVED** [review]
:+1:

- [x] I tested this: 
1. Ran the tests
2. Created a new company and added custom forms and images
3. Deleted the company using the company button
4. Confirmed the data was delete using the code on the testing instructions.
- [x] I read through the code
- [ ] ~~I checked for accessibility issues~~ NA
- [ ] ~~Includes documentation~~~NA

**Pull request merged**